### DISCUSSION TITLE
How to check if its working or not?

### DISCUSSION THREAD

**Discussion by charlesvictor404** [TobiasPankner/Teams-Auto-Joiner#95] [General]
Im a newbie to python and coding. I have done everything according to the things said in this thread, installed all 3 dependencies, and changed the settings accordingly in the config file. I just executed the auto_join.py file but I dont know if its working or running. Is there anyway to know??

Thankyou

**Comment by TobiasPankner** [comment]
Well, what do you see? Did a chrome window open and is it printing stuff in cmd?

**Reply by charlesvictor404** [reply]
Nope. Cmd comes up for a sec a goes away and that's it. Nothing after that

**Reply by TobiasPankner** [reply]
Did you start it like described in the readme?

**Reply by charlesvictor404** [reply]
Yes I did. But you know what, ill try it all again from start just to be sure. Ill ping you

**Comment by ghost** [comment]
(linux㉿kali)-[~/Desktop/Teams-Auto-Joiner]
 python auto_joiner.py                                                1 ⨯
  File "auto_joiner.py", line 22
    browser : webdriver.Chrome = None    
            ^
SyntaxError: invalid syntax

I done know how to solve it I have zero knowledge about this framework. Can anyone help me.

**Reply by TobiasPankner** [reply]
You are probably using an python version before 3.6. Try updating python and rerunning the program.
